client/components/boards/boardsList.js

```diff
@@ -239,34 +239,41 @@ BlazeComponent.extendComponent({
           evt.preventDefault();
         },
         'click .js-clone-board'(evt) {
-          let title = getSlug(ReactiveCache.getBoard(this.currentData()._id).title) || 'cloned-board';
-          Meteor.call(
-            'copyBoard',
-            this.currentData()._id,
-            {
-              sort: ReactiveCache.getBoards({ archived: false }).length,
-              type: 'board',
-              title: ReactiveCache.getBoard(this.currentData()._id).title,
-            },
-            (err, res) => {
-              if (err) {
-                console.error(err);
-              } else {
-                Session.set('fromBoard', null);
-                subManager.subscribe('board', res, false);
-                FlowRouter.go('board', {
-                  id: res,
-                  slug: title,
-                });
-              }
-            },
-          );
-          evt.preventDefault();
+          if (confirm('Are you sure you want to clone this board?')) {
```

Comment:
This is not translatable yet. Examples:

- https://github.com/wekan/wekan/blob/main/imports/i18n/data/en.i18n.json#L308
- https://github.com/wekan/wekan/blob/main/client/components/cards/cardDetails.js#L3
- https://github.com/wekan/wekan/blob/main/client/components/cards/cardDetails.js#L1489